src/condor_tools/htcondor_cli/__init__.py

```diff
@@ -78,6 +79,7 @@
 NOUNS["ap"] = AccessPoint
 NOUNS["cm"] = CentralManager
 
-# Should we knob these lines?
-from htcondor_cli.annex import Annex
-NOUNS["annex"] = Annex
+# annex needs fcntl which does not exist on windows
+if _os_name != 'nt':
+    from htcondor_cli.annex import Annex
+    NOUNS["annex"] = Annex
```

Comment:
It would seem more Pythonic to just try to import the module and ignore if it fails, rather than care about which OS it's running on:

```suggestion
# annex needs fcntl which does not exist on windows
try:
    from htcondor_cli.annex import Annex
    NOUNS["annex"] = Annex
except ImportError:
    pass
```

If you'd prefer to spew an import exception on other platforms if that does fail, that's fine, too.

Reply:
why do work that is guaranteed to fail?